python/tvm/relay/transform/mixed_precision.py

```diff
@@ -40,6 +40,8 @@
     "nn.conv3d_transpose",
     "nn.dense",
     "nn.batch_matmul",
+    "nn.bias_add",
```

Comment:
Not sure if we can change this default list. Better to have another CPU list, otherwise you need to evaluate the impact to NV hardware.